### PR TITLE
Feature/registration screen api

### PR DESCRIPTION
**Pull request by tbd-develop**
- Check out, verify can build
- Verify can run unit tests
- Ask any question in review comments
- Add name to trello card

### CODE REVIEW

**Comment by tbd-develop** [comment]
[Registration Screen API](https://trello.com/c/Ru4IzTjY/8-registration-screen-api)

**Comment by Junicus** [comment]
[Registration Screen API](https://trello.com/c/Ru4IzTjY/8-registration-screen-api)

**Comment by theancientcoder** [comment]
Setting a global.json is not required if one is using Visual Studio even if there are multiple .Net Core SDKs installed. So perhaps we need to clarify for those not using VS to set global.json.

**Comment by Junicus** [comment]
> Setting a global.json is not required if one is using Visual Studio even if there are multiple .Net Core SDKs installed. So perhaps we need to clarify for those not using VS to set global.json.

Instructions on that could work, simple enough.  If that is the way to go then I don't see a problem with the merge.

File: src/persistence/persistence.csproj [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.6" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.2.6" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.6" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.EntityFrameworkCore">
+      <HintPath>..\..\..\..\..\Program Files\dotnet\sdk\NuGetFallbackFolder\microsoft.entityframeworkcore\2.2.0\lib\netstandard2.0\Microsoft.EntityFrameworkCore.dll</HintPath>
```
Comment (canhorn) [inline src/persistence/persistence.csproj]: Why is this like this?

Comment (tbd-develop) [reply src/persistence/persistence.csproj]: Fixed in latest push. VS 2019 quirk?

**Pull request merged**